pkg/driverbuilder/builder/amazonlinux.go

```diff
@@ -66,9 +66,6 @@ rm -Rf /tmp/kernel
 mkdir -p /tmp/kernel
 mv usr/src/kernels/*/* /tmp/kernel
 
-# Change current gcc
-ln -sf /usr/bin/gcc-{{ .GCCVersion }} /usr/bin/gcc
```

Comment:
Dropped the `amazonGccVersionFromKernelRelease` function because it was just returning default GCC version.